### PR TITLE
Documentation: Add Code Climate badges to README.md

### PR DESCRIPTION
**Pull request by beorn7**
@brancz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 [![Docker Repository on Quay](https://quay.io/repository/prometheus/prometheus/status)][quay]
 [![Docker Pulls](https://img.shields.io/docker/pulls/prom/prometheus.svg?maxAge=604800)][hub]
 [![Go Report Card](https://goreportcard.com/badge/github.com/prometheus/prometheus)](https://goreportcard.com/report/github.com/prometheus/prometheus)
+[![Code Climate](https://codeclimate.com/github/prometheus/prometheus/badges/gpa.svg)](https://codeclimate.com/github/prometheus/prometheus)
+[![Issue Count](https://codeclimate.com/github/prometheus/prometheus/badges/issue_count.svg)](https://codeclimate.com/github/prometheus/prometheus)
```
Comment (brancz) [inline README.md]: Looking at the analyzed results, it all seems to come from vendored code. But from my understanding the excluded paths should have taken care of that no?

Maybe @larkinscott has an idea...

**Comment by brancz** [comment]
My comment is rather an issue with the config than these badges. So lgtm 👍

**Comment by beorn7** [comment]
The vendored code is excluded. When it was still included, we scored a GPA of 1.06, but according to the badge and on the site itself, we score 4.0 now. Also the issues all look legit, with the exception of the autogenerated protobuf code in the remote write path. I'll add an exclusion for that one. Stay tuned...

**Comment by larkinscott** [comment]
@beorn7 @brancz It looks like this may have been resolved, but let me know and I can run this past my CX team just to be safe!

**Pull request merged**